### PR TITLE
sound boxes with pics for routines and say

### PR DESCRIPTION
**Pull request by masssamasssa**
- added content boxes in sound.html for routines closing, intervention 1 + 2 as well as say
- texts still need to be added
- pics and audio in google drive

### CODE REVIEW

File: serv/website/sound.html [hunk 1]
```diff
@@ -38,6 +38,108 @@
         </nav> <!-- END OF NAVBAR-EXPAND -->
     </header> <!-- END OF HEADER -->
     <main>
+        <div id="content-box" class="container-fluid">
+            <h2 class="text-uppercase">SOUND AROUND YOU<br> <span class="subheadercb">MERSEBURG</span></h2>
+            <div class="row">
+                <div class="col-xs-12 col-sm-12 col-md-6 col-lg-6 col-xl-6">
+                    <div class="row row-cols-2"> <!-- row-cols-2 -->
+                        <img src="images/say3.jpeg" class="col img-fluid" alt="Workshop">
```
Comment (softbobo) [inline serv/website/sound.html:46]: we should provide image descriptions in the `alt` tags, i.e. to help visually impaired people using screenreaders or to help with seo. an exemplary use here could be 
´´`html
field recording workshop in merseburg
```
 you can read up more [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img#attr-alt)

File: serv/website/sound.html [hunk 2]
```diff
@@ -38,6 +38,108 @@
...
+                        <img src="images/say1.jpeg" class="col img-fluid" alt="Recording">
```
Comment (softbobo) [inline serv/website/sound.html:47]: we use .jpg and .png endings in other places on the website. i think we should strive to only use one raster image format and consistent naming. i suggest `.jpg`

**Review by softbobo — CHANGES_REQUESTED** [review]
good job. i marked a few inconsistencies and try to introduce best practices afaik those in terms of webdesign

File: serv/website/sound.html (hunk 2 above)
Comment (masssamasssa) [reply serv/website/sound.html:47]: Feel free to change code and endings directly on your end to save time with re-upload of files on Google Drive. Sounds good?

Comment (masssamasssa) [reply serv/website/sound.html]: Done inside code. Please change the file endings on the actual files yourself.

File: serv/website/sound.html (hunk 1 above)
Comment (masssamasssa) [reply serv/website/sound.html]: Done

**Pull request merged**